packages/web3-validator/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ErrorObject } from 'ajv';
+import { AbiParameter } from './private_types';
+
+export { JSONSchemaType } from 'ajv';
```

Comment:
```suggestion
import { ErrorObject, JSONSchemaType } from 'ajv';
import { AbiParameter } from './private_types';
```

I feel like eslint should've complained about this